src/nodejs/account/cli/list.js

```diff
@@ -42,25 +42,27 @@ if (program.Debug) {
 
 
 if (program.watch) {
-  var result = account.get_watch_addresses(configData)
+  var accounts = account.get_watch_addresses(configData)
 
-  if (result && result.length)
-  print('result:')
-  result.forEach((r) => {
-    dbg.logDeep('', r)
-  })
+  if (accounts && accounts.length) {
+    print('result:')
+    accounts.forEach((o) => {
+      dbg.logDeep('', o)
+    })
+  }
 } else {
-  var result = account.list(configData)
-
-
-  if (result) {
+  var result
+  var accounts = account.list(configData)
 
+  if (accounts) {
     if (program.name) {
-      json.findAllKeysWhere(result, program.name, (key, val) => {
+      json.findAllKeysWhere(accounts, program.name, (key, val) => {
         result = val
```

Comment:
you're overriding this variable with each loop? This way  only the last loop is stored and all other iterations are lost

Reply:
I'm okay with this for now as there should only be one account by that name. I'll add a note though.